Jenkinsfile

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/groovy
+
+def getTestComponentName() {
+    return 'fh-mbaas'
+}
+
+def getTestComponentConfigs() {
+    def componentConfigs = [:]
+    def componentName = getTestComponentName()
+    componentConfigs[componentName] = [:]
+    componentConfigs[componentName]['gitHubOrg'] = 'feedhenry'
+    componentConfigs[componentName]['repoName'] = 'fh-mbaas'
+    componentConfigs[componentName]['baseBranch'] = 'master'
+    componentConfigs[componentName]['repoDir'] = ''
+    componentConfigs[componentName]['cookbook'] = 'fh-mbaas'
+    componentConfigs[componentName]['buildType'] = 'node'
+    componentConfigs[componentName]['distCmd'] = 'grunt fh:dist'
+    componentConfigs[componentName]['buildJobName'] = "build_any_jenkinsfile"
+    componentConfigs[componentName]['gitUrl'] = "git@github.com:feedhenry/fh-mbaas.git"
+    componentConfigs[componentName]['gitHubUrl'] = "https://github.com/feehdenry/fh-mbaas"
+    return componentConfigs
+}
+
+def testStage(name, body) {
+    stage(name) {
+        step([$class: 'WsCleanup'])
+        print "#### test_${name} ####"
+        body()
+    }
+}
+
+node {
+    env.BRANCH_NAME = env.BRANCH_NAME ?: 'master'
+    String gitref = env.CHANGE_ID ? "pr/${env.CHANGE_ID}" : env.BRANCH_NAME
```

Comment:
If it's a PR, `CHANGE_ID` will be set, otherwise if it's the master (or other branch) job (e.g. after merge), then `BRANCH_NAME` will be set.